task/probe.go

```diff
@@ -1,266 +0,0 @@
-		return nil, fmt.Errorf("error parsing bitrate from track %d: %w", stream.Index, err)
-	}
-	track := &api.AssetTrack{
-		Type:        stream.CodecType,
-		Codec:       stream.CodecName,
-		StartTime:   startTime,
-		DurationSec: duration,
-		Bitrate:     bitrate,
-	}
-	if stream.CodecType == "video" {
-		fps, err := parseFps(stream.AvgFrameRate)
-		if err != nil {
-			return nil, err
-		}
-		track.Width = stream.Width
-		track.Height = stream.Height
-		track.FPS = fps
-		track.PixelFormat = stream.PixFmt
-	} else if stream.CodecType == "audio" {
-		sampleRate, err := strconv.Atoi(stream.SampleRate)
-		if stream.SampleRate != "" && err != nil {
-			return nil, fmt.Errorf("error parsing sample rate from track %d: %w", stream.Index, err)
-		}
-		bitDepth, err := strconv.Atoi(stream.BitsPerRawSample)
-		if stream.BitsPerRawSample != "" && err != nil {
-			return nil, fmt.Errorf("error parsing bit depth (bits_per_raw_sample) from track %d: %w", stream.Index, err)
-		}
-		track.Channels = stream.Channels
-		track.SampleRate = sampleRate
-		track.BitDepth = bitDepth
-	}
-	return track, nil
-}
-
-func parseFps(framerate string) (float64, error) {
-	if framerate == "" {
-		return 0, nil
-	}
-	parts := strings.SplitN(framerate, "/", 2)
-	if len(parts) < 2 {
-		fps, err := strconv.ParseFloat(framerate, 64)
-		if err != nil {
-			return 0, fmt.Errorf("error parsing framerate: %w", err)
-		}
-		return fps, nil
-	}
-	num, err := strconv.Atoi(parts[0])
-	if err != nil {
-		return 0, fmt.Errorf("error parsing framerate numerator: %w", err)
-	}
-	den, err := strconv.Atoi(parts[1])
-	if err != nil {
-		return 0, fmt.Errorf("error parsing framerate denominator: %w", err)
-	}
-
-	if den == 0 {
-		// If numerator and denominator are 0 return 0.0 for the FPS
-		// 0/0 can be valid for a video track i.e. mjpeg
-		if num == 0 {
-			return 0, nil
-		}
-
-		// If only denominator is 0 then the framerate is invalid
-		return 0, errors.New("invalid framerate denominator 0")
-	}
-
-	return float64(num) / float64(den), nil
-}
-
-func logProbeData(assetId, filename string, probeData *ffprobe.ProbeData) {
-	streamFields := []string{}
-	var width, height int
-	var maxStartTime float64
-	for _, stream := range probeData.Streams {
-		add := func(field string, value string) {
-			streamFields = append(streamFields, fmt.Sprintf("stream_%d_%s=%q", stream.Index, field, value))
-		}
-		add("type", stream.CodecType)
-		add("codec", stream.CodecName)
-		add("bitrate", stream.BitRate)
-		add("startTime", stream.StartTime)
-		add("duration", stream.Duration)
-		if stream.CodecType == "video" {
-			add("pixelFormat", stream.PixFmt)
-			width, height = stream.Width, stream.Height
-		}
-		if startTime, err := strconv.ParseFloat(stream.StartTime, 64); err == nil {
-			maxStartTime = math.Max(maxStartTime, startTime)
-		}
-	}
-	glog.Infof("Probed video file assetId=%s filename=%q format=%q width=%d height=%d bitrate=%s startTime=%v maxStreamStartTime=%v %s",
```

Comment:
@mjh1 I guess [this alert](https://eu-metrics-monitoring.livepeer.live/grafana/alerting/LNUr4PKVk/edit?returnTo=%2Falerting%2Flist%3Fsearch%3D) is not needed since this line got removed?

Reply:
Yeah i'd remove, must not be relevant anymore, I'm not aware we're checking for that condition on the alert anymore